### PR TITLE
Bitwarden v0.1

### PR DESCRIPTION
**Pull request by enoch85**
Fixes https://github.com/nextcloud/vm/issues/681

### CODE REVIEW

**Comment by enoch85** [comment]
@nextcloud/vm if you want to review maybe?

I haven't tested it yet.

**Comment by enoch85** [comment]
Tested and works as intended for now --> merge.

File: apps/tmbitwarden.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+If you just want to run Bitwarden locally (not connecting your smartphone) then you can use 'localhost' as domain.
+If you on the other hand want to run this on a domain, then please create a DNS record and point it to this server.
+Well add support for SSL in a later version of this script, but for now you have to do that stuff manually.
+
+The script is based on this documentation: https://help.bitwarden.com/article/install-on-premise/
+It's a good idea to read that before you start this script."
+
+# Test RAM size (2GB min) + CPUs (min 2)
+ram_check 2 Bitwarden
+cpu_check 2 Bitwarden
```
Comment (kspearrin) [inline apps/tmbitwarden.sh]: You should be ok with just 1 CPU.

Comment (enoch85) [reply apps/tmbitwarden.sh]: Yeah for Bitwarden itself, yes. This VM runs more applications than that. 

Thanks for the advice though, keep em coming :)

**Pull request merged**